tests/flashcore/test_session_analytics_gaps.py

```
@@ -0,0 +1,449 @@
+        # Insert cards
+        in_memory_db.upsert_cards_batch(sample_cards)
+
+        manager = ReviewSessionManager(
+            db_manager=in_memory_db,
+            scheduler=FSRS_Scheduler(),
+            user_uuid=uuid4(),
+            deck_name="Math"
+        )
+
+        # GAP: No session start tracking
+        start_time = datetime.now(timezone.utc)
```

Ruff (0.11.9)

Local variable `start_time` is assigned to but never used

Remove assignment to unused variable `start_time`

(F841)